osi_hostvehicledata.proto

```diff
@@ -275,4 +287,205 @@ message HostVehicleData
         //
         optional GeodeticPosition geodetic_position = 3;
     }
+
+    //
+    // \brief State of one automated driving function on the host vehicle.
+    //
+    message VehicleAutomatedDrivingFunction
+    {
+        // The particular function being reported about.
+        //
+        optional FunctionName function_name = 1;
+
+        // Custom function name.
+        //
+        // Only used if function_name is set to FUNCTION_NAME_OTHER.
+        //
+        optional string custom_name = 2;
+
+        // The activation state of the function.
+        //
+        // This is whether the function has actually been triggered, for
+        // example, a warning has been raised, or additional braking is
+        // in effect.
+        //
+        optional ActivationState activation_state = 3;
+
+        // Custom activation state.
+        //
+        // Only used if the activation_state is set to ACTIVATION_STATE_OTHER.
+        //
+        optional string custom_activation_state = 4;
+
+        // Custom detail.
+        //
+        // An opaque set of key-value pairs which capture any user specific
+        // details that may be relevant.  This could include details about
+        // how a warning was raised (dashboard, audible, etc.) or it could
+        // be about settings which would influence evaluation, such as
+        // sensitivity settings.
+        //
+        repeated KeyValuePair custom_detail = 5;
+
+        // A list of possible automated driving features.
+        //
+        // \note This can span (in theory) from Level 0 all the way to Level 5.
+        //
+        // References:
```

Comment:
**CCB Output 12.08.2021**
ToDo:
Check if OSI style guide matches this format

Reply:
CCB 02.02.2022:
Clarification: Topic here is the form of references.
Todo: @stefancyliax

Reply:
Fixed the style of referencing links